src/anchor.js

```diff
@@ -28,7 +29,43 @@ const aggregators = {
             r += part.r
         }
         return new Point(x / len, y / len, r / len)
-    }
+    },
+    intersect: (config, name, parts) => {
+        const get_line_from_point = (point, axis, offset=1000) => {
+            let p1 = [point.x - offset, point.y]
+            let p2 = [point.x + offset, point.y]
+            if(axis == 'y') {
+                p1 = [point.x, point.y - offset]
+                p2 = [point.x, point.y + offset]
+            }
+
+            let line = new m.paths.Line(p1, p2)
+            line = m.path.rotate(line, point.r, point.p)
+
+            return line
+        };
+
+        a.unexpected(
+            config, name,
+            [...aggregator_common, ...['axis1', 'axis2']]
+        )
+        a.in(config.axis1, 'axis1', ['x', 'y'])
+        a.in(config.axis2, 'axis2', ['x', 'y'])
+        a.arr(parts, 'parts', 2, 'Point')
+
+        const line1 = get_line_from_point(parts[0], config.axis1)
+        const line2 = get_line_from_point(parts[1], config.axis2)
+        var intersection = m.path.intersection(line1, line2);
+
+        a.assert(intersection != null, `There is no intersection for ${name}`)
+
+        const intersection_point_arr = intersection.intersectionPoints[0]
+        const intersection_point = new Point(
+            intersection_point_arr[0], intersection_point_arr[1]
```

Comment:
The newly created point should carry on some kind of rotation info from its origins... I would probably vote for averaging the two, so the resulting intersection rotation also makes sense intuitively? See quick sketch.

![image](https://github.com/ergogen/ergogen/assets/213400/1905a9db-5316-42f6-bc48-e5077c4e36eb)

Reply:
I think that a mathematical average of the rotation of the two lines intersecting is not what I would expect to happen to the intersect operation. While it's easy to calculate if I want to undo it, I feel that a better outcome is either:

1  Reset to a default orientation (i.e. zero)
2 Maintain the rotation of the last (or first) point provided

Reply:
I agree with @ceoloide that an averaged rotation would probably not be what people would expect here. 

It could result in arbitrary numbers and would be difficult to "undo", because (if I am not wrong), we can only adjust rotation relativly to the existing rotation. And since the averaged rotation is not something a person will know, it will be difficult to reset it to a desired value.

Maybe a person could use affect to overwrite this behavior, but either way I think this could be a big source of confusion.

In my own use-cases I also don't see where I would ever want it to be averaged. But maybe I am missing something.

But of course, this is your baby and whatever you think is right, I will implement. I just wanted to cast my vote for a rotation of zero.

Just let me know what you prefer.